src/main/java/com/gdsc/pikpet/entity/UserLike.java

```diff
@@ -0,0 +1,18 @@
+package com.gdsc.pikpet.entity;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Getter
+public class UserLike {
+    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+
+    @Setter @ManyToOne @JoinColumn(name = "userId")
```

Comment:
```suggestion
    @Setter @ManyToOne @JoinColumn(name = "user_account_id")
```
칼럼 어노테이션에는 스네이크 케이스(DB 변수명에 주로 사용되는 형식)으로 하는 게 어떠신가요? 
물론 이렇게 해도 DB에는 스네이크 케이스로 변환되어 들어가는데, 이렇게 다르게 해놓으면 헷갈릴 거 같아요
그리고 userId보다는 그냥 Table명_id로 통일하는게 헷갈리지 않을 거 같아요